packages/arb-bridge-eth/contracts/vm/NodeGraph.sol

```diff
@@ -107,23 +107,45 @@ contract NodeGraph is ChallengeType {
         bytes32 logsAccHash;
     }
 
-    function pruneLeaf(
-        bytes32 from,
-        bytes32[] calldata leafProof,
-        bytes32[] calldata latestConfirmedProof
+    function pruneLeaves(
+        bytes32[] memory fromNodes,
```

Comment:
Maybe better to have a single `fromNode` and a single proof from there to `latestConfirmed`, then multiple path proofs to prunable leaves.  The single `fromNode` would just be the oldest ancestor of all of the leaves we are pruning.

Reply:
Not fixing this now.  Creating an issue for this instead.